src/test/mir-opt/nll/region_subtyping_basic.main.nll.0.32bit.mir

```diff
@@ -70,30 +70,30 @@ fn main() -> () {
         StorageLive(_8);                 // bb2[0]: scope 3 at $DIR/region-subtyping-basic.rs:21:9: 21:18
         StorageLive(_9);                 // bb2[1]: scope 3 at $DIR/region-subtyping-basic.rs:21:15: 21:17
         _9 = (*_6);                      // bb2[2]: scope 3 at $DIR/region-subtyping-basic.rs:21:15: 21:17
-        _8 = ConstValue(Scalar(<ZST>): fn(usize) -> bool {use_x})(move _9) -> [return: bb3, unwind: bb7]; // bb2[3]: scope 3 at $DIR/region-subtyping-basic.rs:21:9: 21:18
+        _8 = ConstValue(Zst: fn(usize) -> bool {use_x})(move _9) -> [return: bb3, unwind: bb7]; // bb2[3]: scope 3 at $DIR/region-subtyping-basic.rs:21:9: 21:18
```

Comment:
This looks like something is debug-printing the `ConstValue` variant name...? I am not sure where this is happening.

Reply:
No it's not that fallback.
It might be the fallback in the `ty::Array(..) | ty::Tuple(..) | ty::Adt(..)` branch? That looks not great, it should really fall back to the remaining `match` arms...

Reply:
I give up on this for now, it's not really an worse than before.

Reply:
Aaah I should have checked the test file flags.